src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,33 @@
+.card {
+  &__stars {
+    display: flex;
+    align-items: end;
```

Comment:
Code Style issue: According to BEM methodology, `align-items` property is not usually used with `end` value. It's better to use `flex-end`, `flex-start`, `center`, `baseline`, or `stretch`.